services/provider/interfaces/interfaces.go

```diff
@@ -16,8 +16,12 @@ type StorageClientOnboarding interface {
 	GetOnboardingTicket() string
 	GetConsumerName() string
 	GetClientOperatorVersion() string
+	GetClusterId() string
```

Comment:
I think we should align with what pb generator outputs

Reply:
I meant to change proto from `clusterId` to `clusterID`, asked here because the current func convention isn't consistent `GetClusterId()` and `SetClusterID()`

Reply:
+1 if we are introducing the change in the proto.

Reply:
Annoyingly enough linting fails with the lowercase, so changed to uppercase.